uktrade/data-hub-frontend
# Feature/dashboard toggle

## Description of change

Adds a new toggle component for use on the new personalised dashboard page. There was an existing `ToggleSection` component, but it was not styled as needed for the new dashboard. The old style can now be chosen by specifying the appearance prop as "secondary", with the new (default) style as "primary". This allows us to use the primary style for the dashboard and use the more subtle secondary component on things like the investment projects list filters.

## Test instructions

Go to the new personalised dashboard page by toggling the feature flag. The Reminders component and the Investment Project Summary components should appear inside togglable sections. Note that the Reminders component has a red top border (to indicate high significance) above it, with the Project Summary showing a blue top border.

`ToggleSection` can also be seen on storybook with the primary and secondary appearances.

Please check that the new component is consistent with the [designs](https://glc39r.axshare.com/#id=ozm45m&p=spec_reminders&g=1) and toggles open and closed.

## Screenshots
### Before

![Screenshot from 2021-03-16 16-45-44](https://user-images.githubusercontent.com/1234577/111347413-17980100-8677-11eb-9fa0-8b14dd2d213f.png)

### After

![Screenshot from 2021-03-16 16-34-49](https://user-images.githubusercontent.com/1234577/111347091-c6880d00-8676-11eb-9b24-2c19ac053fa0.png)

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
